docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3"
+services:
+  #PHP Service
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: digitalocean.com/php
+    container_name: app
+    restart: unless-stopped
+    tty: true
+    environment:
+      SERVICE_NAME: app
+      SERVICE_TAGS: dev
+    working_dir: /var/www
+    volumes:
+      - ./:/var/www
+    networks:
+      - app-network
+
+  #Nginx Service
```

Comment:
Nginx build configs.
What's good to pay attention to is, the `volumes` block! I would advise you to read about this block and what it means, but basically it will save files between your local device and the docker image. If you delete the image, the data stays in your local machine _PC_